src/app/views/settings/Settings.tsx

```diff
@@ -81,6 +82,14 @@ function Settings(props: ISettingsProps) {
 
     if (authenticated) {
       menuItems.push(
+        {
+          key: 'switch-user-app-mode',
+          text: messages[permissionModeType ? "Use Explorer as sample Teams application" : "Use Explorer as logged-in user"],
+          iconProps: {
+            iconName: permissionModeType ? "TeamsLogo" : "Contact",
```

Comment:
Should "TeamsLogo" and "Contact" be a constant?

Reply:
All the other iconName properties seem to be hard-coded strings, e.g. lines 125 and 133 in `Profile.tsx`